src/Build.UnitTests/Evaluation/ItemEvaluation_Tests.cs

```diff
@@ -332,6 +332,78 @@ public void ItemOperationsShouldExpandIndirectItemReferences(string projectConte
         }
 
         [Fact]
+        public void OnlyPropertyReferencesGetExpandedInPropertyFunctionArgumentsInsideIncludeAttributes()
+        {
+            var projectContent =
+@"<Project>
+        <ItemGroup>
+            <A Include=`1;$(P)` Exclude=`$(P)`/>
```

Comment:
Why have the `Exclude` here? I am thinking that whatever `$(P)` would evaluate to (like some incorrect value due to a bug) then it won't affect `@(A)`, so doesn't that make this a no-op, in a sense?